docs/examples/MarrmotM01.ipynb

```diff
@@ -2,196 +2,142 @@
  "cells": [
```

Comment:
<p>We should use <code>cfg_dir</code> instead of <code>work_dir</code> for public API.</p><p> </p><p>Also can we print the <code>model</code> to get an overview</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eWaterCycle/ewatercycle/pull/158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>